src/vector/index.html

```diff
@@ -14,6 +14,7 @@
     <link rel="apple-touch-icon" sizes="180x180" href="<%= require('../../res/vector-icons/apple-touch-icon-180x180.png') %>">
     <link rel="manifest" href="manifest.json">
     <link rel="shortcut icon" href="<%= require('../../res/vector-icons/favicon.ico') %>">
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/katex@0.10.2/dist/katex.min.css" integrity="sha384-yFRtMMDnQtDRO8rLpMIKrtPCD5jdktao2TV19YiZYWMDkUR5GQZR/NOVTdquEx1j" crossorigin="anonymous">
```

Comment:
We'd like to avoid linking to external CDNs like this. Can we add `katex` as a dependency and somehow require the file, such as `require('katex/dist/katex.min.css')`?

Reply:
oh sure, done!